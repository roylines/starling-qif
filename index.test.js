const {toQIF} = require('.');

describe('toQIF', () => {
  it('should create qif', () => {
    const trans = require('./index.test.json');
    const qif = toQIF(trans);
    expect(qif).toMatchSnapshot();
  });
});
