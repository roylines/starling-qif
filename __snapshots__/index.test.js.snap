// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toQIF should create qif 1`] = `
"!Type:Bank
D2020/01/28
T-45.00
PR Grimes
MNew Hat
^
D2020/01/27
T118.00
PNegan
Mwww.negan.com\\\\GREATER LONDON HOUSE\\\\NW1 7FB GBR
^
D2020/01/27
T-39.99
PSanctuary Marketplace
MSCTYMktplace sanctuary.co.uk GBR
^
"
`;
